### ISSUE TITLE
DJI Air 2S BMS BQ9003 error  code: 110024/30074, Project DOUBLE BATTERY

### ISSUE THREAD

**Issue by alanzhch** [o-gs/dji-firmware-tools#325]
Does anybody now if the new battery of the air 2s, has a "lock sensor" which recognize if the battery is in the drone or not?
The old dji mavic air batterys had a lock sensor!

Fact: I took the cellpack out of the case and tried to start the drone with only the battey pack inside. The drone starts but takeoff is blocket with the error codes in the titel of that request.

If the project works i will share the documentery how tho build a double battery, out of two original packs.

**Comment by alanzhch** [comment]
FYI: I use a CP2112 with DJI Battery Killer, no PF Flag active at this moment.

**Comment by akhilzid** [comment]
May be the drone itself triggering the error, air 2 can detect that the battery is properly inserted, just short the terminals near the battery lock to cheat that.

**Comment by alanzhch** [comment]
Cheers, i guess you mean the two holes where the metal clips snap in from the battery case, in to the drone case?
If you mean something else with "the terminals" let me know please.

I ordered two new batterycases because i had to cut the originals of, to take the cellpack out.. :) 
Now i try to solder a parallel connection between the two cellpack from the most positive and most negative poles.
The pack which goes in to the drone as before will be charged with the original charger and the upper pack with a charger via xt60 connection. 
The two pack will be connected via xt60 as well and be balanced as before.

**Comment by alanzhch** [comment]
The company from singapoore who sells double batterys, dont export the to other nations...
So i have to do it by myself..

**Comment by akhilzid** [comment]
This is the "lock sensor" in air 2S shorting this two terminals will detect battery.
![Uploading Screenshot_2023-02-12-00-08-55-55_965bbf4d18d205f782c6b8409c5773a4.jpg…]()

**Comment by alanzhch** [comment]
Ahhh one problem solved for the moment. Thank you! 
I will be back next week and will try it out.

**Comment by Aleksancho1304** [comment]
what is your current progress?

**Comment by horstmr** [comment]
One of the double terminals of the battery lock sensor on the left side has broken in my drone. Do the sensors on both sides have to be shorted so it can take off? 
The drone is currently not with me so I can test this.
Thanks!

**Comment by alanzhch** [comment]
Yes, otherwise you will get the "battery installation error" :) Try to solder another pin to it
